bindings/rust/src/lib.rs

```diff
@@ -107,6 +108,21 @@ impl Drop for KZGSettings {
     }
 }
 
+impl Blob {
+    pub fn from_bytes(bytes: &[u8]) -> Result<Self, Error> {
```

Comment:
Where do we use these methods? If we use them just for tests, can we perhaps make this clear?

In particular, perhaps at this point it would be a good idea to split tests to a different file and move all these helpers there too? I think it will be nice to know which functions are security-related (and need to be audited) and which functions are only used in tests.

Reply:
Unless there's some reason for not allowing the user of the lib to create a `Blob` from a slice, it's a useful function to expose to the user imo.

Reply:
Good point, because `Blob` is part of the public interface.

However, objects like `KZGCommitment` and `KZGProof` are not part of the public interface, so I don't think a user of the lib would need to create a KZGCommitment or KZGProof from bytes.

Reply:
By the way, this is used when loading a blob from a test file. I also think it's best to keep it.